Patch for GNU/kFreeBSD portability from Petr Salinger
<Petr.Salinger@t-systems.cz>.

Kerberos RT #3466
Debian bug #261712

=== krb5/src/appl/telnet/telnet/sys_bsd.c
==================================================================
--- krb5/src/appl/telnet/telnet/sys_bsd.c	(revision 2038)
+++ krb5/src/appl/telnet/telnet/sys_bsd.c	(local)
@@ -43,6 +43,8 @@
 #include <sys/types.h>
 #include <sys/time.h>
 #include <sys/socket.h>
+#include <sys/ioctl.h>
+
 #ifdef HAVE_SYS_SELECT_H
 #include <sys/select.h>
 #endif
=== krb5/src/appl/telnet/telnetd/defs.h
==================================================================
--- krb5/src/appl/telnet/telnetd/defs.h	(revision 2038)
+++ krb5/src/appl/telnet/telnetd/defs.h	(local)
@@ -38,6 +38,7 @@
  */
 #include <sys/types.h>
 #include <sys/param.h>
+#include <sys/ioctl.h>
 
 #ifndef	BSD
 # define	BSD 43
